notebooks/community/prediction/custom_prediction_routines/SDK_Triton_PyTorch_Local_Prediction.ipynb

```diff
@@ -0,0 +1,1829 @@
+{
```

Comment:
<p>is there a webdoc link you can add about the parameter choices?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/661/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
I can't find a good doc for this. Attached a quickstart guide.